### PR TITLE
Batanoff update readme

### PR DESCRIPTION
**Pull request by batanoffs**
updated readme and yarn.lock (versions)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +31,11 @@ yarn add ra-auth-ui
 Run the Demo
 
 ```sh
-npm run start-demo
-#or
-yarn start-demo
+yarn install 
+cd ./examples/demo
+yarn install
+yarn dev
```
Comment (ValentinnDimitroff) [inline README.md:37]: Actaully we want to still run `yarn start-demo` but just add a note to install the node modules the first time before running the project

File: examples/demo/yarn.lock
Comment (ValentinnDimitroff) [inline examples/demo/yarn.lock:1]: Let's reset this file, we don't want to upgrade dependencies here

**Review by ValentinnDimitroff — CHANGES_REQUESTED** [review]
Please, also modify the `start-demo` script in the `package.json` to call `yarn dev` instead of `yarn start`

File: examples/demo/yarn.lock
Comment (batanoffs) [reply examples/demo/yarn.lock:1]: Added new commit where i revert the changes

**Review by ValentinnDimitroff — APPROVED** [review]

**Comment by ValentinnDimitroff** [comment]
Are we gonna merge this one @batanoffs  :)

**Pull request merged**